src/deb/control/postinst

```diff
@@ -6,7 +6,7 @@ set -e
 . /usr/share/debconf/confmodule
 
 # defaults file
-CONFIG="/etc/default/@PACKAGE@"
+CONFIG="/etc/default/[[name]]"
```

Comment:
can you please verify if this location is in sync with one on line 93 and default conf location `jmxtrans.sh` looks for